noggin/form/sync_token.py

```diff
@@ -0,0 +1,24 @@
+from flask_wtf import FlaskForm
+from wtforms import StringField, PasswordField
+from wtforms.validators import DataRequired, Optional
+
+
+class SyncTokenForm(FlaskForm):
+    username = StringField(
+        'Username', validators=[DataRequired(message='You must provide a user name')]
+    )
+
+    password = PasswordField(
+        'Password', validators=[DataRequired(message='You must provide a password')]
+    )
+
+    first_code = PasswordField(
+        'First OTP', validators=[DataRequired(message='You must provide a first code')]
+    )
+
+    second_code = PasswordField(
```

Comment:
I wonder if it's useful to have a `PasswordField` here, since the tokens are one-time only. On most websites with OTP support that I've seen, I think the token field was clear text. What do you think?

Reply:
sure, that makes sense. pushed now!